portfolio.html

```diff
@@ -43,65 +43,57 @@
+          <a href="/portfolio" class="link db dtc-m dtc-l bg-white pa2 br2 relative shadow-6 mb4 mb0-ns">
+            <p class="pa2 bb b--black dwyl-dark-gray pb3 tl-ns montserrat f5 mt2 pb4-l">More than 250,000 artefacts, made accessible
+              and searchable!</p>
+            <img src="./img/common/science-museum-logo.jpg" alt="Science Museum logo" class="w-50 w-80-m center db pb3"/>
           </a>
-        </div>
-        <div class="w-10-ns pt2 ph2 pb3 dib-ns dtc-ns v-top tc center">
-          <a href="/portfolio"  class="link dwyl-dark-gray">
-            <div class="pa2 bg-white br2 h5 h6-l shadow-2">
-              <p class="pa2 bb b--black pb3 tl-ns montserrat f6">Automating data collection means our TA's
-                can get on with what really matters.</p>
-              <img src="./img/common/inclusive-classrooms-logo.png" alt="Inclusive Classrooms logo" class="w-80 w-75-l pt3 pt3-l"/>
-            </div>
+          <a href="/portfolio" class="link db dtc-m dtc-l bg-white pa2 br2 relative shadow-6 mb4 mb0-ns">
+            <p class="pa2 bb b--black dwyl-dark-gray pb3 tl-ns montserrat f5 mt2">Automating data collection means our TA's
+              can get on with what really matters.</p>
+            <img src="./img/common/inclusive-classrooms-logo.png" alt="Inclusive Classrooms logo" class="w-60 w-80-m pt2 pt2-l pb3 center db"/>
           </a>
-        </div>
-        <div class="w-10-ns pt2 ph2 pb3 dib-ns dtc-ns v-top tc center">
-          <a href="/portfolio" class="link dwyl-dark-gray">
-            <div class="pa2 bg-white br2 h5 h6-l shadow-2">
-              <p class="pa2 bb b--black pb3 tl-ns montserrat f6">We increased community engagement with an
-              exclusive social network.</p>
-              <img src="./img/common/ellen-macarthur-foundation-logo.png" alt="Ellen MacArthur Foundation logo" class="w-90 w-90-l pt4 pt4-l"/>
-            </div>
+          <a href="/portfolio" class="link db dtc-m dtc-l bg-white pa2 br2 relative shadow-6">
+            <p class="pa2 bb b--black dwyl-dark-gray pb3 tl-ns montserrat f5 mt2">We increased community engagement with an
+            exclusive social network.</p>
+            <img src="./img/common/ellen-macarthur-foundation-logo.png" alt="Ellen MacArthur Foundation logo" class="w-70 w-90-m pt3 pt3-l pb3 center db"/>
           </a>
         </div>
       </section>
     </section>
     <section class="bg-white pt5 pb6">
-      <section class="mw9 center ph3">
-        <h2 class="pl3 pr3 f2 mt0 w-100 tc tl-ns dwyl-dark-gray">Open Source</h2>
+      <section class="mw8 center ph3">
+        <h2 class="pl3 pr3 f1 mt0 w-100 tc tl-ns dwyl-dark-gray">Open Source</h2>
         <p class="pl3 pr3 f4 b ph0-m dwyl-dark-gray pb3">We also create and maintain over 200 open source repositories on <a href="https://github.com/dwyl" class="dwyl-teal-dark link" >GitHub</a>, check out some of our most popular ones below:</p>
```

Comment:
This is a very random use of the dark-teal colour:
<img width="531" alt="screen shot 2017-10-09 at 19 34 52" src="https://user-images.githubusercontent.com/4185328/31353054-f1696148-ad28-11e7-93ed-83e0ce31944e.png">